### ISSUE TITLE
Experiments on cityscapes dataset

### ISSUE THREAD

**Issue by dipamgoswami** [arthurdouillard/CVPR2021_PLOP#26]
Hello @arthurdouillard, I enjoyed reading your paper. I have a few queries regarding the implementation using the cityscapes dataset.
1. I am reproducing the experiments for [plop_cityscapesDomain_1-1.sh](https://github.com/arthurdouillard/CVPR2021_PLOP/blob/main/scripts/cityscapesDomain/plop_cityscapesDomain_1-1.sh) .
I see that the method is specified as FT and not PLOP. I am unable to understand why that is the case here? Why is the method not PLOP? 
2. There is no explicit background class in the cityscapes dataset. How is that dealt with in plop implementation? It may not be an issue in domain-incremental but will be required in class-incremental experiments. I see one argument 'dont_predict_bg' but that hasn't been used. 

Let me know if you can clarify these doubts. Thanks.

**Comment by arthurdouillard** [comment]
Hey,

Thank you for your interest in my work!

1. Because they are slight difference in domain-incremental, we don't need pseudo-labeling (nor UNCE for MiB), so I set by hand the options of PLOP, see https://github.com/arthurdouillard/CVPR2021_PLOP/blob/main/scripts/cityscapesDomain/plop_cityscapesDomain_1-1.sh#L19 
2. In the domain-incremental cityscapes there are no class background, and all classes are known from the start. But you're right about the class-incremental cityscapes: I tackle this problem in my follow-up paper https://arxiv.org/abs/2106.15287 (whose code isn't yet released). In this case, I consider as background all pixels from past and future classes, but it's kind of a "fake" background. The argument `dont_predict_bg` is because multiple papers (including MiB & ILT if I'm not mistaken), see the background (the true one) as unknown (255) during test-time and thus doesn't consider it. Which I've found unrealistic as in a real-situation you cannot discard background pixels, especially in my situation where I eval the model after each step while MiB&ILT only eval at the last step.

Is that clearer? :)
